### ISSUE TITLE
Issue with Bitbar

### ISSUE THREAD

**Issue by hanj0415** [kevinjalbert/notion-toolbox#20]
I have below errors from bitbar.
```
Traceback (most recent call last):
  File "/Volumes/Home/rickhan/notion-toolbox/bitbar/get_current_tasks.py", line 17, in <module>
    message = "Working on Nothing | href=%s" % app_url(api.current_day().get_browseable_url())
  File "/Volumes/Home/rickhan/Library/Python/3.7/lib/python/site-packages/cachetools/__init__.py", line 46, in wrapper
    v = func(*args, **kwargs)
  File "/Volumes/Home/rickhan/notion-toolbox/shared/notionscripts/notion_api.py", line 65, in current_day
    day_name = current_date.strftime(self.config.custom_day_format())
TypeError: strftime() argument 1 must be str, not None
```

**Comment by kevinjalbert** [comment]
@hanj0415 so a recent change was made that allowed the day page formatting to be customizable (#16). The error you are seeing suggests that `self.config.custom_day_format()` is evaluating to `None`. I'm curious what you have for your `config.json` for that option?

https://github.com/kevinjalbert/notion-toolbox/blob/549f763d936c3337e008973d79afafb1fd122ad0/alfred/src/config.py#L52-L53

The above snippet shows that a default value will be used if you don't specify the option (the default is what we had prior to the new change).

**Comment by hanj0415** [comment]
![image](https://user-images.githubusercontent.com/59560208/84518494-bcae3380-ad0b-11ea-9fb0-2ff89bbd1f47.png)
config.json is attached which has new change.

**Comment by kevinjalbert** [comment]
@hanj0415 yeah, so the `CUSTOM_DAY_FORMAT` should be a valid https://strftime.org/ format.

If you don't have any specific needs for a different format, I'd just remove this config from your file and try again.

**Comment by hanj0415** [comment]
@kevinjalbert I've removed ```CUSTOM_DAY_FORMAT``` from config.json, but I had the same error as below.
```
❯ /Volumes/Home/rickhan/Dropbox/Bitbar/show_current_tasks.sh
Traceback (most recent call last):
  File "/Volumes/Home/rickhan/notion-toolbox/bitbar/get_current_tasks.py", line 17, in <module>
    message = "Working on Nothing | href=%s" % app_url(api.current_day().get_browseable_url())
  File "/Volumes/Home/rickhan/Library/Python/3.7/lib/python/site-packages/cachetools/__init__.py", line 46, in wrapper
    v = func(*args, **kwargs)
  File "/Volumes/Home/rickhan/notion-toolbox/shared/notionscripts/notion_api.py", line 65, in current_day
    day_name = current_date.strftime(self.config.custom_day_format())
TypeError: strftime() argument 1 must be str, not None
```

**Comment by kevinjalbert** [comment]
@hanj0415 I was able to reproduce the issue on my end (it is when there was no current task in place).

I've pushed a fix (#21). Update, then close and restart BitBar. It should be working after that.

Reopen the issue if it doesn't work still.
